test/NotEmptyTest.php

```diff
@@ -27,6 +27,33 @@ public function setUp()
         $this->validator = new NotEmpty();
     }
 
+    public function testConstructorWithTypeArray()
+    {
+        $validator = new NotEmpty([
+            'php',
```

Comment:
Could you replace php|boolean example with Integer|Zero I think those values represent better the bitmask behavior.